search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -13,20 +13,23 @@
 // the License.
 
 /**
- * Displays the information modal when a country on the map is clicked.
+ * Displays the information modal when a region on the map is clicked.
  * @param {?google.maps.MouseEvent} e Click event.
  */
-function onClickCountry(e) {
-  if (e.feature.getProperty('country_data') != null) {
-    $('#region-info-modal').modal('show');
-
-    // Update Modal with information for relevant country.
-    const countryName = e.feature.getProperty('name');
-    const countryId = e.feature.getId();
-    document.getElementById('modal-title').innerText = countryName;
-    displayTopResultsForCurrentTrend(countryId);
-    setCountryTrends(countryId);
+function onClickRegion(e) {
+  // Don't display anything when the map is on state level, or when the clicked
+  // country has no data.
```

Comment:
Wait I thought on the state level they can click to view the top trends in that state?

Reply:
Sadly no. I just realized the trends api can only get trends for countries, not states

Reply:
Wait so what can we view in US mode then?

Reply:
Only the popularity scores for each state. We can talk about this tmr